Add Vscode Github plugin

vinokurig opened the pull request:
https://github.com/eclipse/che/issues/11867

benoitf commented on v3/plugins/ms-vscode/vscode-git/latest/meta.yaml:
```diff
@@ -0,0 +1,15 @@
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/microsoft/vscode/tree/master/extensions/git
+category: Other
+firstPublicationDate: '2016-03-11'
```
hello, AFAIK it's the date published on this registry no ?

vinokurig replied:
I thought that it is the first commit date of the related repository, will fix that to the date of merge.

benoitf commented on v3/plugins/ms-vscode/vscode-github-pullrequest/latest/meta.yaml:
```diff
@@ -0,0 +1,15 @@
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/microsoft/vscode-pull-request-github
+category: Other
+firstPublicationDate: '2018-06-27'
```
same small remark there

vinokurig replied:
will fix that to the date of merge.

benoitf commented:
@vinokurig could you please also reference https://github.com/eclipse/che/issues/11867 in the description so we're sure it's part of 'must fix' PR

azatsarynnyy commented:
These two plugins have `latest` versions without the corresponded particular versions (e.g. 1.0.0).
As I understand, `latest` should always correspond to some particular plugin version (copying it's `meta.yaml`).
Should the PR also add the concrete plugins' versions?

vinokurig commented:
@azatsarynnyy Fixed

benoitf commented:
Hello, I'm thinking as git is quite a "default" extension, it should be included in the default che-theia image and not be part of the registry

benoitf commented:
could be hooked in https://github.com/eclipse/che-theia/blob/master/dockerfiles/theia/Dockerfile#L85-L86 and https://github.com/eclipse/che-theia/blob/master/dockerfiles/theia/Dockerfile#L108

vinokurig commented:
@benoitf 
> Hello, I'm thinking as git is quite a "default" extension, it should be included in the default che-theia image and not be part of the registry

The VsCode git plugin is built from vscode build scripts that are located in the vscode repository, can we include the `vsix` file to the che-theia build?

benoitf commented:
@vinokurig AFAIK yes
you just use curl or wget to grab it from https://github.com/che-incubator/vscode-git/releases

vinokurig commented:
@benoitf
* Added the vscode git plugin to che-theia (https://github.com/eclipse/che-theia/pull/328)
* Updated the PR (removed the git extension from plugins)

benoitf commented:
@vinokurig thanks
